recipe_batches/recipe_batches.py

```diff
@@ -1,14 +1,47 @@
 #!/usr/bin/python
+'''
+Write a function that receives a recipe in the form of a dictionary, as well as all of the ingredients you have available to you, also in the form of a dictionary. Both of these dictionaries will have the same form, and might look something like this:
+
+```python
+{
+  'eggs': 5,
+  'butter': 10,
+  'sugar': 8,
+  'flour': 15
+}
+```
+[Notes]
+
+Need to take in 2 dict: one = recipe, other = what you gotz
+
+Compare what you gotz to the recipe, see if basic cookies can be made
+
+If so, see how many batches can be made. **Make sure you use the key with the lowest amount of batches as answer**
+
+'''
 
 import math
 
+
 def recipe_batches(recipe, ingredients):
-  pass 
+    store = []
+
```

Comment:
wait, be careful here. check this again. if you have more ingredients than the recipe requires, for example, you have on hand eggs, milk, flour, but the recipe only calls for milk and flour, then you can still make the dish the recipe calls for! double check this logic! :D